eng/Version.Details.xml

```diff
@@ -29,33 +29,33 @@
       <Uri>https://github.com/aspnet/AspNetCore-Tooling</Uri>
       <Sha>e3da5266f5c8b3595fc6863aa549d5217479fdb2</Sha>
     </Dependency>
-    <Dependency Name="dotnet-ef" Version="5.0.0-alpha.1.19564.1">
+    <Dependency Name="dotnet-ef" Version="5.0.0-alpha.1.19564.5">
```

Comment:
These changes look bizarre. Should be no EF packages w/ 5.0 versioning in the 'release/3.1' branch. So, why do these changes show up as differences from what's in 'master'?